package.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "dm3",
+    "private": true,
     "workspaces": {
```

Comment:
What is the reason to make it private? I think that'll lead to issues when the package is about to be published to npm

Reply:
[This is how a monorepo should be set up.](https://classic.yarnpkg.com/lang/en/docs/workspaces/), including the "private" setting. It prevents the monorepo itself being published by mistake, which it never should. Instead, the individual workspaces should be published, which is still possible.